mix.exs

```diff
@@ -54,15 +54,17 @@ defmodule Membrane.MP3.Lame.Mixfile do
 
   defp deps do
     [
-      {:membrane_core, "~> 0.8.0"},
-      {:membrane_caps_audio_raw, "~> 0.5.0"},
+      {:membrane_core, "~> 0.9.0"},
```

Comment:
This element also qualifies for auto-demand introduction

Reply:
Good point. Let's introduce it with #35 and I will rebase this PR before merging.